gguf-py/gguf/tensor_mapping.py

```diff
@@ -136,6 +138,7 @@ class TensorNameMap:
             "model.layers.{bid}.ln1",                               # rwkv7
             "model.layers.{bid}.input_layernorm",                   # llama4
             "transformer_encoder.{bid}.attention_norm",             # neobert
+            "blk.{bid}.attn_norm",                                  # smoldocling
```

Comment:
I think this `blk.*` naming is wrong. It should be the HF safetensors naming scheme, not the GGUF naming scheme